app/views/about/accessibility.en.html.erb

```diff
@@ -0,0 +1,157 @@
+<%# CONTNENT NEEDS CREATING %>
+
+<% title 'Privacy policy' %>
```

Comment:
Are all pages privacy policy content, on purpose? 🤣 

If it is a placeholder, wouldn't be better to have at least their title correspond with what the page is suppose to be, and then as body something like "Placeholder. Copy to be decided." or similar?

Having all them the same is confusing.

Reply:
👍 Good idea. Done!